src/components/TodoComponents/Todo.js

```diff
@@ -4,7 +4,12 @@ import './Todo.css';
 export default class Todo extends React.Component {
     render() {
         return (
-            <div id={this.props.id}>{this.props.task}</div>
+            <div 
+                className={`task ${this.props.completed === true ? 'done' : ''}`} 
```

Comment:
Here we need to make sure that `this.props.completed` exists so that the `done` class is added. We do this by passing a "completed" prop to this component. 👍